docs/en/current/settings/account.md

```diff
@@ -215,24 +223,28 @@ include the original message by clicking the *Quote message* button in the messa
 a message.
 
 ### Strip signatures on reply
-When this is checked K-9 Mail will try to detect the correspondents signature and remove it when quoting the original 
+When this is checked K-9 Mail will try to detect the correspondent's signature and remove it when quoting the original 
 message.
 
+### Upload sent messages
+
+Upload sent messages to your account's Sent folder if the send was successful.
+
 ### Outgoing server
 
-Please see [Configuring the outgoing server](../accounts/add.md#configuring-the-outgoing-server) for the available 
+Please see [Configuring the outgoing server](../accounts/outgoing.md#configuring-the-outgoing-server) for the available 
 settings.
 
 
 ## Folders
 
 ### Auto-expand folder
-The folder you specify here will be opened when you click on an account in the account list. Set to *-NONE-* when 
+The folder you specify here will be opened when you click on an account in the account list. Set to *NONE* when 
 clicking an account should open the folder list.
```

Comment:
@cketti I'm not sure if this is still true. I think if you select NONE, it defaults to Inbox. The folder list disappeared in the 5.8 redesign didn't it?

Reply:
The behavior has changed slightly. Starting the app will try the following (in this order):
- open Unified Inbox if configured
- open auto-expand folder of first account if configured
- open Inbox of first account if available
- open Outbox of first account (if the folder list hasn't been retrieved from the server yet, this is the only folder we know exists locally) - I'd omit this information from the documentation

When the auto-expand folder is set to *None*, opening an account via the drawer will leave the drawer open and display the folder list of that account, allowing the user to select a folder right away. If auto-expand is to a specific folder the drawer is closed and the message list for that folder is displayed.